extensions/objects/templates/image-with-regions-editor.component.spec.ts

```diff
@@ -177,6 +237,9 @@ describe('ImageWithRegionsEditorComponent', () => {
   it('should return selected region trash icon css styling' +
   ' when a region is selected', () => {
     component.selectedRegion = 0;
+    spyOn(contextService, 'getEntityType').and.returnValue(
+      AppConstants.ENTITY_TYPE.EXPLORATION);
+    spyOn(contextService, 'getExplorationId').and.returnValue('exploration_id');
```

Comment:
Do we have to use this here? Can you check in other tests also if we have to `spyOn` contextService?

Reply:
Done